docs/content/guides/dagster/branch_depl_with_snowflake.mdx

```diff
@@ -0,0 +1,325 @@
+---
+title: Branch Deployments with Snowflake | Dagster
+description: This guide illustrates a workflow that enables testing Dagster code in your cloud environment without impacting your production data.
+---
+
+# Branch Deployments with Snowflake
```

Comment:
Maybe too late in the game to open up a title discussion, but to me "Branch Deployments with Snowflake" makes me think we are using Snowflake to make Branch Deployments work, rather than creating fresh dbs for branch deployments to use for testing.  I'm having a hard time coming up with alternativesI like though. 

"Using Dagster to create snowflake dbs for cloud branch deployments" 
"Using Dagster Cloud Branch Deployments with Isolated Snowflake Databases for Testing" 

all of these seem really wordy to me